### PR TITLE
Add `usbip` example

### PR DESCRIPTION
**Pull request by sosthene-nitrokey**
This makes it faster to test with usbip than using the nitrokey-3-firmware usbip runner.

### CODE REVIEW

File: examples/usbip.rs [hunk 1]
```diff
@@ -0,0 +1,74 @@
+// Copyright (C) 2022 Nitrokey GmbH
+// SPDX-License-Identifier: CC0-1.0
+
+//! USB/IP runner for opcard.
+//! Run with cargo run --example --features apdu-dispatch (and optionally rsa4096-gen)
```
Comment (robin-nitrokey) [inline examples/usbip.rs:5]: Outdated comment.

File: examples/usbip.rs [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        vid: VID,
+        pid: PID,
+    };
+    trussed_usbip::Builder::new(virt::Ram::default(), options)
+        .dispatch(CoreOnly)
+        .init_platform(move |platform| {
+            let ui: Box<dyn trussed::platform::UserInterface + Send + Sync> =
+                Box::new(UserInterface::new());
+            platform.user_interface().set_inner(ui);
+        })
```
Comment (robin-nitrokey) [inline examples/usbip.rs:71]: Why do we need this?  Isn’t this just duplicating the user interface?

Comment (sosthene-nitrokey) [reply examples/usbip.rs:71]: 🤔

I copied this from opcard. This probably comes from earlier versions of pc-usbip-runner?

It should probably also be removed from opcard.

Comment (robin-nitrokey) [reply examples/usbip.rs:71]: Maybe it comes from the usbip runner where we actually use a custom `UserInterface` in `inner` to display messages etc.  But in this case, it is effectively a no-op.

Comment (sosthene-nitrokey) [reply examples/usbip.rs]: Good catch.

File: examples/usbip.rs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    let options = trussed_usbip::Options {
+        manufacturer: Some(MANUFACTURER.to_owned()),
+        product: Some(PRODUCT.to_owned()),
+        serial_number: Some("TEST".into()),
+        vid: VID,
+        pid: PID,
+    };
+    trussed_usbip::Builder::new(virt::Ram::default(), options)
+        .dispatch(CoreOnly)
+        .init_platform(|_platform| {})
```
Comment (robin-nitrokey) [inline examples/usbip.rs:67]: I think you can just remove these two lines because these are the default settings.

**Review by robin-nitrokey — APPROVED** [review]

**Comment by robin-nitrokey** [comment]
Generally, I’d rather improve the nk3 usbip runner instead of adding usbip examples to all applications to reduce maintenance efforts, so let me know if there is a specific issue with it, other than having to patch fido-authenticator and its dependencies.

**Comment by sosthene-nitrokey** [comment]
Right, but having the runner available just over the app I'm working on is a preferable than having to patch and run in another repo to debug the FIDO app. This is especially valid  when breaking changes are made to fido-authenticator.

**Pull request merged**